enumerables.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    x = 0
+    selfitem = self
+    while x < selfitem.size
+      yield(selfitem[x])
+      x += 1
+    end
+  end
```

Comment:
- [ ] Enumerables my_each returns an Enumerator if no block is given
     Failure/Error: expect(array.my_each).to be_an(Enumerator)